docs/development_guide/docker_style_guide.md

```diff
@@ -0,0 +1,82 @@
+# Docker Style Guide
+This document will provide a guide for writing Dockerfiles for the pipelines.  
+The guide will provide a list of best practices for writing Dockerfiles, and will also 
+provide a list of common mistakes to avoid. The guide is for those wanting to contribute
+Dockerfiles to the pipelines repository.
+
+
+## Dockerfiles Organization
+All docker related resources should be placed in the `docker` directory of the repository. The `docker` directory
+contains a subdirectory for each Dockerfile and its related resources. The subdirectory name should 
+start with an abbreviation of data type the docker tool will process followed by the name of the tool.
+For example, the docker image for the `bwa` aligner that will process long reads would be placed in the `docker/lr-bwa` directory. 
+
+## Docker Subdirectory Folder
+Each Docker subdirectory should contain the following files and folders:
+- `Dockerfile`: The Dockerfile for the Docker image.
```

Comment:
I think the list formatting is not showing up correctly on the webpage.

Reply:
https://broadinstitute.github.io/long-read-pipelines/development_guide/docker_style_guide/#docker-subdirectory-folder

Reply:
fixed